### DISCUSSION TITLE
Does this support SRNE?

### DISCUSSION THREAD

**Discussion by chukaonline** [davidrapan/ha-solarman#324] [Q&A]
Hi, I was wondering if SRNE is supported and if so which profile should be used. If it is not supported, any plans in future?
i have SRNE HESP48120S200

**Comment by davidrapan** [comment]
Hi @chukaonline, I guess the only what you can find out is if you try it. It can be just some rebranded devices and thus could work with already existing profiles or you will have to contact your manufacturer so they provide you with modbus rtu documentation. 😉
